examples/digitalidentity/app/Http/Controllers/IdentityController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Support\Facades\Log;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Yoti\DigitalIdentityClient;
+use Yoti\Identity\Policy\PolicyBuilder;
+use Yoti\Identity\ShareSessionRequestBuilder;
+use Yoti\YotiClient;
+
+class IdentityController extends BaseController
+{
+    public function show(DigitalIdentityClient $client)
+    {
+        try {
+
+            $policy = (new PolicyBuilder())
+                ->withFamilyName()
+                ->withGivenNames()
+                ->withFullName()
+                ->withDateOfBirth()
+                ->withGender()
+                ->withNationality()
+                ->withPhoneNumber()
+                ->withSelfie()
+                ->withEmail()
+                ->withDocumentDetails()
+                ->withDocumentImages()
+                ->build();
+
+            $redirectUri = 'https://host/redirect/';
+
+            $shareSessionRequest = (new ShareSessionRequestBuilder())
+                ->withPolicy($policy)
+                ->withRedirectUri($redirectUri)
+                ->build();
+
+            $session = $client->createShareSession($shareSessionRequest);
+
+            $createdQrCode = $client->createShareQrCode($session->getId());
+
+            $fetchedQrCode = $client->fetchShareQrCode($createdQrCode->getId());
+
+            $sessionFetched = $client->fetchShareSession($session->getId());
```

Comment:
These lines are not required, as Webshare handles them automatically.

Reply:
removed

Reply:
removed